controls/sysctl_spec.rb

```diff
@@ -194,6 +194,15 @@
   end
 end
 
+control 'sysctl-17a' do
+  impact 1.0
+  title 'Disable log martians'
+  desc 'log_martians can cause a denial of service attack to the host'
+  describe kernel_parameter('net.ipv4.conf.default.log_martians') do
```

Comment:
@bitvijays cool! Suggestion: put this `describe` to the `sysctl-17` above, its more or less the same option (`all` vs `default`)